exams/management/commands/test_authorization_user_exam.py

```diff
@@ -0,0 +1,148 @@
+
+        cls.command = authorization_user_exam.Command()
+
+    def test_bulk_exam_authorization(self):
+        """For all users in any program"""
+        self.command.handle("authorization_user_exam", all=True)
+        for user in self.users:
+            self.assertTrue(ExamProfile.objects.filter(profile=user.profile).exists())
+            self.assertTrue(ExamAuthorization.objects.filter(
+                user=user,
+                course=self.course_run.course
+            ).exists())
+
+    def test_bulk_exam_authorization_given_user_name(self):
+        """For given user"""
+        user1 = self.users[0]
+        user2 = self.users[1]
+        self.command.handle("authorization_user_exam", username=user1.username)
+
+        # user 1 is authorized for exam
+        self.assertTrue(ExamProfile.objects.filter(profile=user1.profile).exists())
+        self.assertTrue(ExamAuthorization.objects.filter(
+            user=user1,
+            course=self.course_run.course
+        ).exists())
+
+        # user 2 is not authorized
+        self.assertFalse(ExamProfile.objects.filter(profile=user2.profile).exists())
+        self.assertFalse(ExamAuthorization.objects.filter(
+            user=user2,
+            course=self.course_run.course
+        ).exists())
+
+    def test_bulk_exam_authorization_given_program(self):
+        """For given program id"""
+        self.command.handle("authorization_user_exam", program_id=self.program.id)
+        for user in self.users:
+            self.assertTrue(ExamProfile.objects.filter(profile=user.profile).exists())
+            self.assertTrue(ExamAuthorization.objects.filter(
+                user=user,
+                course=self.course_run.course
+            ).exists())
+
+    def test_bulk_exam_authorization_given_wrong_program(self):
+        """For given program id"""
+        with self.assertRaises(CommandError) as ce:
+            self.command.handle("authorization_user_exam", program_id=-1)
+
+        assert ce.exception.args[0] == '[Exam authorization] Invalid program id: %s', -1
+        for user in self.users:
+            self.assertFalse(ExamProfile.objects.filter(profile=user.profile).exists())
+            self.assertFalse(ExamAuthorization.objects.filter(
+                user=user,
+                course=self.course_run.course
+            ).exists())
+
+    def test_bulk_exam_authorization_given_wrong_user(self):
+        """For given program id"""
+        with self.assertRaises(CommandError) as ce:
+            self.command.handle("authorization_user_exam", username="invalid_user")
+
+        assert ce.exception.args[0] == '[Exam authorization] Invalid username: %s', "invalid_user"
+        for user in self.users:
+            self.assertFalse(ExamProfile.objects.filter(profile=user.profile).exists())
+            self.assertFalse(ExamAuthorization.objects.filter(
+                user=user,
+                course=self.course_run.course
+            ).exists())
```

Comment:
I think that these tests are not necessary: the only thing to be done to test the management command is to assert that when the command is called, the right backend function is called.
So you can mock the entire backend function and just assume that it works.